code/web/src/modules/crate/Item.js

```diff
@@ -29,6 +29,9 @@ class Item extends PureComponent {
   }
 
   onClickSubscribe = (crateId) => {
+    if (!this.props.user.details.style) {
+      return this.props.history.push(userRoutes.survey.path)
+    }
```

Comment:
When the subscribe button is pressed on a crate, this directs the user to the style survey if no user style exist.

Reply:
this makes my life easy! good work!